jawa/constants.py

```diff
@@ -506,7 +575,26 @@ def pack(self, fout):
                     constant._name_index,
                     constant._descriptor_index
                 ))
-
+            elif isinstance(constant, ConstantMethodHandle):
+                            write(pack(
```

Comment:
It looks fine in raw on GitHub, I've noticed there was a blank line missing after the else if block so I've added that on. Let me know if anything else is missing.